oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBStorageClient.java

```diff
@@ -56,7 +56,7 @@ public class BanyanDBStorageClient implements Client, HealthCheckable {
     private final DelegatedHealthChecker healthChecker = new DelegatedHealthChecker();
 
     public BanyanDBStorageClient(String host, int port) {
-        this.client = new BanyanDBClient(host, port);
+        this.client = new BanyanDBClient(host + ":" + port);
```

Comment:
We should change storage configurations with docs to accept multiple IP:port(s) for cluster server.

Reply:
Fixed. Now a comma-separate string is supported